packages/editor/src/components/post-featured-image/index.js

```diff
@@ -87,6 +102,23 @@ function PostFeaturedImage( { currentPostId, featuredImageId, onUpdateImage, onR
 						/>
 					</MediaUploadCheck>
 				}
+				{ ! featuredImageId &&
+					<div>
```

Comment:
Hi @Jackie6 I guess this div can be removed right?

Reply:
@jorgefilipecosta The div has been removed.